Source/WebCore/page/ResizeObservation.cpp

```diff
@@ -52,24 +52,26 @@ void ResizeObservation::updateObservationSize(const BoxSizes& boxSizes)
     m_lastObservationSizes = boxSizes;
 }
 
-auto ResizeObservation::computeObservedSizes() const -> BoxSizes
+std::optional<ResizeObservation::BoxSizes> ResizeObservation::computeObservedSizes() const
```

Comment:
You can write this for greater brevity:
```
auto ResizeObservation::computeObservedSizes() const -> std::optional<BoxSizes> 
```
I suppose it’s a matter of taste which one is better.

Reply:
I do like that notation better.